optimum/exporters/onnx/utils.py

```diff
@@ -131,15 +128,12 @@ def get_decoder_models_for_export(
         `Dict[str, Tuple[Union[`PreTrainedModel`, `TFPreTrainedModel`], `OnnxConfig`]: A Dict containing the model and
         onnx configs for the encoder and decoder parts of the model.
     """
-    models_for_export = dict()
+    models_for_export = {}
 
```

Comment:
nit: The function docstring is incorrect.

Reply:
Solved